hit-on.cabal

```diff
@@ -81,4 +81,20 @@ executable hit
   ghc-options:         -Wall
                        -threaded
                        -rtsopts
-                       -with-rtsopts=-N
\ No newline at end of file
+                       -with-rtsopts=-N
+
+
+test-suite hit-on-test
+  type:                exitcode-stdio-1.0
```

Comment:
We have `common-options` stanza. You can import it as a first line of your `test-suite` stanza :slightly_smiling_face: 

After doing this you can remove some redundant code:

1. `-Wall` from `ghc-options`
2. `LANGUAGE` pragmas from your `Main.hs` file (they are already included into `common-options`)

Reply:
This makes a lot of sense. Making the change!

Reply:
Due to the import of `common-options`, I got a warning due to `[-Wmissing-export-lists]` so I added the `main` function to the export list at the start of the module (module name declaration).